developerportal/apps/staticbuild/wagtail_hooks.py

```diff
@@ -147,11 +144,7 @@ def _static_build_async(self, force=False, pipeline=settings.STATIC_BUILD_PIPELI
                 logger.info(f"{log_prefix} Created temporary build dir {build_dir}")
 
             for name, command in pipeline:
-                if settings.DEBUG and not force:
-                    logger.info(
-                        f"{log_prefix} (wagtail-bakery) command '{name}' skipped."
-                    )
-                else:
+                if not settings.DEBUG:
```

Comment:
I swapped the order of these to make it easier to read by putting the core case first, not second